.github/workflows/rust.yaml

```
@@ -1,57 +1,29 @@
-      - name: Install cargo-llvm-cov
-        run: cargo binstall cargo-llvm-cov
-      - name: Install Nextest
-        run: cargo binstall cargo-nextest
-      - name: Run tests with coverage
-        run: cargo llvm-cov nextest
-      - name: Upload test results to Codecov
-        if: ${{ !cancelled() }}
-        uses: codecov/test-results-action@47f89e9acb64b76debcd5ea40642d25a4adced9f # v1
+
+      - name: Restore Dagger cache
+        uses: actions/cache@v3
```

actionlint (1.7.7)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)